src/configurations/destinations/bq/ui-config.json

```diff
@@ -132,6 +132,60 @@
           },
           "required": false,
           "footerNote": "Note: Please specify time in UTC"
+        },
+        {
+          "type": "singleSelect",
+          "immutable": true,
+          "label": "Partition Column",
+          "value": "partitionColumn",
+          "options": [
+            {
+              "name": "Ingestion Time",
+              "value": "_PARTITIONTIME"
+            },
+            {
+              "name": "Loaded At",
+              "value": "loaded_at"
+            },
+            {
+              "name": "Received At",
+              "value": "received_at"
+            }
```

Comment:
If `_PARTITIONTIME` is not picked, the customer can theoretically use any column for partitioning right? As long as they create the table on their end. 

Is there a reason we allow them to select between `loaded_at` and `received_at` ?

Also, we could be a bit more descriptive here, what's the difference between loaded_at and _PARTITIONTIME ?

Reply:
Yeah.